vivarium/core/composition.py

```diff
@@ -394,7 +394,9 @@ def test_composer() -> Dict:
                 'GLC': 0,
                 'MASS': 3,
                 'DENSITY': 10}}}
-    return simulate_composer(toy_compartment, settings)
+    if return_value:
+        return simulate_composer(toy_compartment, settings)
+    return None
```

Comment:
Why all of these changes with `return_value`?

Reply:
`pytest` warned me that all tests should return `None`. It said non-compliant tests would return errors in future versions of `pytest`. We use our test functions in many places where return values are expected. I added the `return_value` option so that values are returned only in these places and not when the tests are run by `pytest`.